src/routes/brands/updateBrand.ts

```diff
@@ -25,14 +27,13 @@ const handler: RouteHandler<UpdateBrandRoute> = async (req, reply) => {
     const brandId = req.params.id;
```

Comment:
Destructuring id with an alias...
```suggestion
const {id:brandId} = req.params;
```